### ISSUE TITLE
activities.3.currency ... does not match with currency of TICKR

### ISSUE THREAD

**Issue by netcarlos** [agusalex/ghostfolio-sync#28]
I'm getting

`2025-03-07 23:00:37,120 - SyncIBKR - INFO - Failed to create: {"error":"Bad Request","message":["activities.3.currency (\"EUR\") does not match with currency of NVDA (\"USD\")"]}`

How could I troubleshoot this error?

**Comment by netcarlos** [comment]
I had to map it to a different ticker
